### DISCUSSION TITLE
I can't get all products on Amazon SP API. How can I do this?

### DISCUSSION THREAD

**Discussion by FatmaSedaOZYURT** [amzn/selling-partner-api-models#2830] [General] [answered]
Hi everybody,
I can't get all products. I searched but I didn't understand this flow.
I read this [page ](https://developer-docs.amazon.com/sp-api-blog/docs/manage-your-product-listings-lifecycle-with-selling-partner-api). Unfortunatelly, I couldn't do it.
Where is that I overlook?
I'm waiting for your answer.
Thanks in advance.

**Comment by stefnats** [comment]
Do you really want to obtain data on all products (ASINs) that are listed on Amazon?

Or what do you want to do exactly?

**Reply by FatmaSedaOZYURT** [reply]
Thank you for response @stefnats.
Actually, My purpose, list all my products in my market with SP API without sku.
But, I guess already all products on Amazon. So mean, my market's products are on Amazon therefore I can't list right?

I'm confused. Can you shed light on this subject?

**Comment by FatmaSedaOZYURT** [comment]
Is there throw light on this subject?
How can I list Amazon products?
I guess it is impossible with Selling Partner API. 🤔 
Can it make with Business API?

**Comment by elifaydemir** [comment]
i have the same problem, no answer?

**Reply by shenglingshi** [reply]
no direct api .you need use report api download listing data to localdb,then show localdb data.

**Reply by NA-Dev** [reply]
> no direct api .you need use report api download listing data to localdb,then show localdb data.

@shenglingshi Any idea how to get the listing report for Vendor accounts? I get an error that the report type is not supported for vendors. I need a list of all my account's ASINs, including bundles.

**Comment by FatmaSedaOZYURT** [comment, marked as answer]
I understood you @shenglingshi 
You suggest me that is should been used Inventory reports. 
I had thought that is useful Business API. 

Thank you for comment.

**Comment by haoxi911** [comment]
[getInventorySummaries](https://developer-docs.amazon.com/sp-api/docs/fbainventory-api-v1-reference#getinventorysummaries) will return a list of FBA SKUs, then [searchCatalogItems](https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-reference#searchcatalogitems) will return the details of each SKU.
